Traceback (most recent call last):
  File "C:\Users\Mark\AppData\Roaming\Python\Python310\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\Mark\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Mark\AppData\Roaming\Python\Python310\site-packages\jupyter_core\utils\__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Program Files\Python310\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\Users\Mark\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Mark\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\Mark\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
#(X â†’ F+[[X]-X]-F[-FX]+X), (F â†’ FF)

axiom = 'X'

n = int(input("Adja meg az iterÃ¡ciÃ³k szÃ¡mÃ¡t: "))

for i in range (n+1):
    new = ''
    print(str(i)+". iterÃ¡ciÃ³: "+ axiom)
    for letter in axiom:
        if letter == 'X':
            new += 'F+[[X]-X]-F[-FX]+X'               
        elif letter == 'F':    
            new += 'FF'
        else:
            new += letter        
    axiom = new
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [1;32mIn[2], line 5[0m
[0;32m      1[0m [38;5;66;03m#(X â†’ F+[[X]-X]-F[-FX]+X), (F â†’ FF)[39;00m
[0;32m      3[0m axiom [38;5;241m=[39m [38;5;124m'[39m[38;5;124mX[39m[38;5;124m'[39m
[1;32m----> 5[0m n [38;5;241m=[39m [38;5;28mint[39m([38;5;28;43minput[39;49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mAdja meg az iterÃ¡ciÃ³k szÃ¡mÃ¡t: [39;49m[38;5;124;43m"[39;49m[43m)[49m)
[0;32m      7[0m [38;5;28;01mfor[39;00m i [38;5;129;01min[39;00m [38;5;28mrange[39m (n[38;5;241m+[39m[38;5;241m1[39m):
[0;32m      8[0m     new [38;5;241m=[39m [38;5;124m'[39m[38;5;124m'[39m

File [1;32m~\AppData\Roaming\Python\Python310\site-packages\ipykernel\kernelbase.py:1250[0m, in [0;36mKernel.raw_input[1;34m(self, prompt)[0m
[0;32m   1248[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[0;32m   1249[0m     msg [38;5;241m=[39m [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[1;32m-> 1250[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(msg)
[0;32m   1251[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[0;32m   1252[0m     [38;5;28mstr[39m(prompt),
[0;32m   1253[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[0;32m   1254[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[0;32m   1255[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[0;32m   1256[0m )

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.

